meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.0.1.bb

```diff
@@ -0,0 +1,25 @@
+require mender-artifact.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender-artifact.git;protocol=https;branch=3.0.x"
+
+# Tag: 3.0.1
+SRCREV = "62b933bd1780ff1ae71e3f62a9642901ff335000"
+
+# Enable this in Betas, not in finals.
+# Downprioritize this recipe in version selections.
+DEFAULT_PREFERENCE = "-1"
```

Comment:
We still keep this as we cannot update the major release in stable thud branch. However, maybe we could adjust the comment to make it clearer?

Reply:
Yes, good idea. For the original recipe as well.